Mergin/plugin.py

```diff
@@ -262,6 +265,46 @@ def configure(self):
             self.on_config_changed()
             self.show_browser_panel()
 
+    def show_no_workspaces_dialog(self):
+        msg = (
+            "Workspace is a place to store your projects and share them with your colleagues. "
+            "Click on the button below to create one. \n\n"
+            "A minimum of one workspace is required to use Mergin Maps."
+        )
+        msg_box = QMessageBox(QMessageBox.Critical, "You do not have any workspace", msg, QMessageBox.Close)
+        create_button = msg_box.addButton("Create workspace", msg_box.ActionRole)
```

Comment:
There can be EE deployment where creation of new workspaces is prohibited (https://ee.dev.merginmaps.com/config).
If that is a case only mergin super user can create new workspaces.

Reply:
Is this something we should handle in the plugin though? What should we do in that case? Skip the *create workspace* button and append *Contact your administrator* to the message?

Reply:
Not sure, server will return 403. I can add some error message there. So if you just display that message, might be good to keep it.

Reply:
ok, I'll change the url to `/workspaces` (now it's pointing to `/) then. If the user sees a 403 he will contact his admin anyway! :)

Reply:
changed url to `/workspaces`